### DISCUSSION TITLE
Why does Ctrl-C not terminate the app when the form is used as as a BubbleTea model

### DISCUSSION THREAD

**Discussion by jbcpollak** [charmbracelet/huh#273] [Q&A] [answered]
I was experimenting with the `examples/scroll`, and how it would behave with the alt screen renderer. I noticed if I replaced `form.Run()` with:

```
//form.Run()
p := tea.NewProgram(form, tea.WithAltScreen())
if _, err := p.Run(); err != nil {
	log.Fatal(err)
}
```

when I press Ctrl-C I get a blank screen, and when I switch to a different terminal, I see that `./scroll` is still running. Killing it from the other terminal restores the terminal. I get the same result when I comment out the `tea.WithAltScreen()` but still use `tea.NewProgram()`, so it doesn't seem to be related to the rendering mode.

However, Ctrl-C works as expected and the app quits when I do the following (instead of the above):

```
form := huh.NewForm(
   // define form here
).WithHeight(5).WithProgramOptions(tea.WithAltScreen())
form.Run()
```

What is the difference between these two modes of execution?

**Comment by maaslalani** [comment, marked as answer]
Hi, this is intentional as when running the form in bubble tea you may not want to quit the app if the user presses ctrl+c, when running the form in stand alone mode however you would want to cancel the form. This is controlled here:

https://github.com/charmbracelet/huh/blob/main/form.go#L558-L561

**Comment by jbcpollak** [comment]
Hum, thanks! 

That makes sense though I'd humbly suggest that could be better documented somewhere.
